css/home.css

```diff
@@ -2,217 +1,170 @@
+body,
+html {
   margin: 0;
   padding: 0;
+  width: 100%;
   overflow-x: hidden;
-  font-family: fantasy;
 }
 
-header {
-  display: grid;
-  grid-template-columns: auto auto;
-  grid-template-rows: auto auto;
-}
-
-.logo {
-  display: grid;
-  grid-row: 1 / 3;
-  overflow: hidden;
+.head {
+  position: absolute;
 }
 
-.logo a {
-  justify-self: center;
-  align-self: center;
+.top-layer {
+  z-index: 2;
 }
 
-.nav-1 {
-  display: grid;
-  justify-content: end;
-  align-content: center;
-  grid-template-columns: auto auto auto auto;
-  grid-gap: 15px;
-  min-height: 50px;
+.dropdown-menu.head {
+  transform: translateY(-5px);
 }
 
-.nav-1 a {
-  text-decoration: none;
+.flag-icon {
+  height: 20px;
 }
 
-.nav-1 a img {
-  width: 20px;
+.bg-custom {
+  transform: translateY(20px);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5398690)